### PR TITLE
Remove slash from end of breakpoint path

### PR DESCRIPTION
**Pull request by KrzysztofSikoraCodete**
screen

https://github.com/jupyterlab/debugger/issues/408
@afshin

### CODE REVIEW

**Comment by jtpio** [comment]
Does this still support the truncation at the beginning of the string with the ellipsis (`...`), as added in https://github.com/jupyterlab/debugger/pull/334?

Or does it undo this change?

**Comment by KrzysztofSikoraCodete** [comment]
@jtpio After my change, support of (...) was unfortunelty removed.
I dived into the subject (directional: rtl) and i found some solution. <bdi> </bdi> tags.

How it works now you can see below.

bidi

**Comment by jtpio** [comment]
Test failure looks similar to a previous issue with the browser check running on GitHub Actions:

- https://github.com/jupyterlab/extension-cookiecutter-ts/pull/53
- https://github.com/jupyterlab/jupyterlab/pull/7813
- https://github.com/actions/virtual-environments/issues/307

File: src/breakpoints/body.tsx [hunk 1]
```diff
@@ -88,6 +88,13 @@ const BreakpointCellComponent = ({
   breakpoints: IDebugger.IBreakpoint[];
   model: BreakpointsModel;
 }) => {
+  const removeFirstCharIfSlash = (breakpoint: IDebugger.IBreakpoint) => {
+    breakpoint.source.path =
+      breakpoint.source.path[0] === '/'
```
Comment (jtpio) [inline src/breakpoints/body.tsx:93]: It would be interesting to see how this behaves with Windows path, since they don't start with a leading `/`.

Comment (jtpio) [reply src/breakpoints/body.tsx:93]: And I guess we still we want to keep the leading `/` and not remove it.

Comment (KrzysztofSikoraCodete) [reply src/breakpoints/body.tsx:93]: Peek 2020-04-17 13-10
Peek 2020-04-17 12-17

Third solution. Now path is not changed.

**Comment by jtpio** [comment]
Thanks @KrzysztofSikoraCodete, this looks like a good solution at the moment to handle this.

The leading `/` is most likely treated as punctuation, and is then moved to the "right" with the `rtl` css property.

Also we will probably revisit the breakpoint UI at some point too: https://github.com/jupyterlab/debugger/issues/171

**Comment by jtpio** [comment]
Strangely enough, the browser check on CI passed on this more recent PR: https://github.com/jupyterlab/debugger/pull/426.

**Comment by KrzysztofSikoraCodete** [comment]
@jtpio Nice to heard that. I found on stackover flow the similar problem with direction `rtl`.
The reason of moved "/" to right side is what as you mentioned.

**Comment by afshin** [comment]
Sweet! Cheers @KrzysztofSikoraCodete!

**Pull request merged**